### ISSUE TITLE
Change Image Size on Text Cell

### ISSUE THREAD

**Issue by marionava** [HappenApps/Quiver#324]
It would be great if I could change the image size on text cells.

**Comment by cjrd** [comment]
+1 I have some super large images in my text cells that look a bit silly =)

**Comment by loriculberson** [comment]
I agree with @cjrd . But I have a workaround for you. Whenever I have to paste an image into Quiver:
1. Paste the image into a Google doc
2. Shrink it
3. Copy and paste it into quiver
